### PR TITLE
docs(readme): Simplified Chinese translation for README

### LINKED ISSUE #18: Translate README.md to Chinese

### PR DESCRIPTION
**Pull request by dreamjz**
# Description
Summarize the change.

Fixes #18 

# Testing
N/A

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [x] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: README_ZH_CN.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# SigLens
+
+SigLens 是一个开源可观测性解决方案，比 Splunk 和 Elastic 高效 **100 倍**。
```
Comment (asiL-tcefreP) [inline README_ZH_CN.md:3]: I think maybe "SigLens 是一个开源的，能够解决可观测性问题的方案，它比 ..." will be better

File: README_ZH_CN.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+## 为什么选择 SigLens：
+为 10,000 多名工程师提供可观测性工具的经验教会了我们一些事情：
+
+- 开发人员不得不在各种的工具之间切换以查看 logs、metrics 和 traces
+- Splunk、DataDog、NewRelic 的费用非常昂贵
+- ElasticSearch 需要太多的机器，集群维护比较困难
+- Grafana Loki 的查询性能较慢
+
+凭借几十年的监控领域经验，我们着手开始构建一个全新的，专为 logs、metrics 和 traces 而设计的，并且具有**零外部依赖**的可观测性数据库。只需一个**单一的二进制文件**，就可以在您的笔记本电脑上运行，处理每天高达 8 TB 的数据。
```
Comment (asiL-tcefreP) [inline README_ZH_CN.md:13]: I think maybe "它只需要一个**单一的二进制文件**，就可以在您的笔记本电脑上运行，并且能够每天处理高达 8 TB 的数据。" will be better

File: README_ZH_CN.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+## 加入我们的 Slack 社区
+在 [Slack](https://www.siglens.com/slack) 上和我们打个招呼吧 👋
+
```
Comment (asiL-tcefreP) [inline README_ZH_CN.md:17]: I guess you might need to add two line break characters `<br />` to align with the original document.

File: README_ZH_CN.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+SigLens 后端独立部署于 UI。 为了允许 UI 连接到后端，可以使用 docker network
+
+```
+wget "https://github.com/siglens/siglens/releases/download/${SIGLENS_VERSION}/server.yaml"
+docker pull siglens/siglens:${SIGLENS_VERSION} 
+mkdir data
+docker run -it --mount type=bind,source="$(pwd)"/data,target=/siglens/data \
+    --mount type=bind,source="$(pwd)"/server.yaml,target=/siglens/server.yaml \
+    -p 8081:8081 -p 80:80 siglens/siglens:${SIGLENS_VERSION}
+```
```
Comment (asiL-tcefreP) [inline README_ZH_CN.md:57]: I think you might need to prepend four spaces to each line for indentation, as the original document has done the same.

File: README_ZH_CN.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+为了能够在重新启动后查询数据，请设置 `server.yaml` 中的 `ssInstanceName` 
+
+`data` 目录挂载的目标路径应和 `server.yaml` 中的 `dataPath` 配置相同。
+
+# 特点：
+
+1. 支持多种数据引入格式(Ingestion formats)：Open Telemetry、Elastic、Splunk HEC、Loki
+2. 支持多种查询语言：Splunk SPL、SQL 和 Loki LogQL
+3. 架构简单，易于上手
```
Comment (asiL-tcefreP) [inline README_ZH_CN.md:67]: I think using ',' instead of '、' may be better.

File: README_ZH_CN.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+#### 创建警报(Creating Alerts)
+
+![Creating Alerts](./static/assets/readme-assets/alerting.png)
+
+#### 实时追踪(Live Tail)
+
+![Live Tail](./static/assets/readme-assets/live-tail.png)
+
+#### Minion Searches
```
Comment (asiL-tcefreP) [inline README_ZH_CN.md:105]: It seems like you forget to translate this line.

**Review by asiL-tcefreP — COMMENTED** [review]
Overall, it's good. Just need to address some minor issues.

**Comment by dreamjz** [comment]
Thank you for your review and sorry for my mistakes.

### Modification Description
- Modified some sentences based on the review
- Added the missing `<br></br>` and indentation to to align with the original document.
- Use `,` to separate English terms

**Comment by asiL-tcefreP** [comment]
Looks pretty good.

**Review by sunitakawane — APPROVED** [review]

**Review by sonamgupta21 — APPROVED** [review]

**Comment by dreamjz** [comment]
### Description 
- Update Chinese doc to align with English doc

**Review by sunitakawane — APPROVED** [review]

**Pull request merged**